### PR TITLE
BUG/CI: Fix doc build ci, fix mysql container over HDD and fix doc section inside Makefile 

### LINKED ISSUE #2027: CI/BUG: LinuxBuildDocs failing
Initially reported at https://github.com/ibis-project/ibis/pull/2024#issuecomment-550143737

```
Traceback (most recent call last):
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/cmd/build.py", line 284, in build_main
    app.build(args.force_all, filenames)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/application.py", line 337, in build
    self.builder.build_update()
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/builders/__init__.py", line 326, in build_update
    len(to_build))
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/builders/__init__.py", line 339, in build
    updated_docnames = set(self.read())
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/builders/__init__.py", line 445, in read
    self._read_serial(docnames)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/builders/__init__.py", line 467, in _read_serial
    self.read_doc(docname)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/builders/__init__.py", line 511, in read_doc
    doctree = read_doc(self.app, self.env, self.env.doc2path(docname))
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/io.py", line 323, in read_doc
    pub.publish()
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/docutils/core.py", line 218, in publish
    self.apply_transforms()
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/docutils/core.py", line 199, in apply_transforms
    self.document.transformer.apply_transforms()
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/transforms/__init__.py", line 90, in apply_transforms
    super().apply_transforms()
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/docutils/transforms/__init__.py", line 172, in apply_transforms
    transform.apply(**kwargs)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/transforms/__init__.py", line 414, in apply
    self.app.emit('doctree-read', self.document)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/application.py", line 440, in emit
    return self.events.emit(event, self, *args)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/sphinx/events.py", line 76, in emit
    results.append(callback(*args))
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/releases/__init__.py", line 612, in generate_changelog
    doctree.walk(changelog_visitor)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/docutils/nodes.py", line 142, in walk
    if child.walk(visitor):
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/docutils/nodes.py", line 142, in walk
    if child.walk(visitor):
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/docutils/nodes.py", line 134, in walk
    visitor.dispatch_visit(self)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/docutils/nodes.py", line 1912, in dispatch_visit
    return method(node)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/releases/__init__.py", line 595, in visit_bullet_list
    releases, _ = construct_releases(node.children, self.app)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/releases/__init__.py", line 502, in construct_releases
    stripped_entries[index + 1:], manager
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/releases/__init__.py", line 429, in handle_upcoming_major_release
    version = Version(obj.number)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/releases/models.py", line 14, in __init__
    super(Version, self).__init__(version_string, partial)
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/semantic_version/base.py", line 102, in __init__
    raise ValueError("Call either Version('1.2.3') or Version(major=1, ...).")
ValueError: Call either Version('1.2.3') or Version(major=1, ...).

Exception occurred:
  File "/opt/conda/envs/ibis-env/lib/python3.6/site-packages/semantic_version/base.py", line 102, in __init__
    raise ValueError("Call either Version('1.2.3') or Version(major=1, ...).")
ValueError: Call either Version('1.2.3') or Version(major=1, ...).
The full traceback has been saved in /tmp/sphinx-err-izxs725u.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!

##[error]Bash exited with code '2'.
##[section]Finishing: Build docs
```

### PR DESCRIPTION
**Pull request by xmnlab**
In this PR:

- fix documentation build CI issue related to `semantic_version` (Resolves #2027)
- fixed doc section inside Makefile
- skip mysql tzinfo  (see: https://github.com/docker-library/mariadb/issues/262)

### CODE REVIEW

**Comment by scottcode** [comment]
@xmnlab Thanks for the work on all these different issues! This addresses a lot of very different and independent issues. Should we consider splitting this up into separate pull requests divided into more cohesive chunks. For example:

1. Doc fixes (Makefile and semantic_version pin)
2. MySQL-related fixes
3. updated omniscidb image

**Comment by xmnlab** [comment]
@scottcode  it sounds good.
but I think that skip mysql tzinfo should be here .. because without that I cannot test it locally.

**Comment by scottcode** [comment]
> @scottcode it sounds good.
> but I think that skip mysql tzinfo should be here .. because without that I cannot test it locally.

Makes sense, @xmnlab. Thanks for splitting out the things that doc build fix doesn't depend on.

**Comment by scottcode** [comment]
Since I don't write access to the project, my approval doesn't enable the merge. Someone else with write privs will need to approve and merge.

File: ci/requirements-docs.yml [hunk 1]
```diff
@@ -10,3 +10,5 @@ pyarrow>=0.12
 sphinx=2.0.1
 sphinx-releases
 sphinx_rtd_theme
+# https://github.com/ibis-project/ibis/issues/2027
```
Comment (jreback) [inline ci/requirements-docs.yml]: what is pulling this in now?

Comment (jreback) [reply ci/requirements-docs.yml]: i would create an issue to remove this pin when the underlying issue is resolved.

Comment (scottcode) [reply ci/requirements-docs.yml]: `sphinx-releases` has an issue that a change in `semantic_version` caused. You're right, we should have another issue to remove the pin when it gets fixed.

Comment (scottcode) [reply ci/requirements-docs.yml]: I just created an issue to unpin `semantic_version` once the issue is fixed (#2033)

Comment (xmnlab) [reply ci/requirements-docs.yml]: thanks @scottcode ! I adding a release note now here.

Comment (jreback) [reply ci/requirements-docs.yml]: @xmnlab ok, lgtm. ping when ready.

File: docs/source/release.rst [hunk 2]
```diff
@@ -8,6 +8,7 @@ Release Notes
    notes for pre-1.0 versions of ibis can be found at :doc:`/release-pre-1.0`
 
 * :release:`1.2.1 <pending>`
+* :bug:`2030` Fix LinuxBuildDocs issue related to semantic_version, fix doc section inside Makefile and skip mysql tzinfo on CI
```
Comment (jreback) [inline docs/source/release.rst]: from a user point of view this is not very useful. what IS useful and worth mentioning, is that we now have a pin in the docs build (its not really user, but developer user here)

Comment (xmnlab) [reply docs/source/release.rst]: sounds good. I updated the release note. let me know if it needs any other change.

**Comment by xmnlab** [comment]
green :)

**Comment by jreback** [comment]
thanks @xmnlab

**Comment by jreback** [comment]
rebase the other PRs and we'll get em in

**Pull request merged**